.clang-tidy

```diff
@@ -44,5 +55,4 @@ readability-container-size-empty,
 HeaderFilterRegex: '^(c10/(?!test)|torch/csrc/(?!deploy/interpreter/cpython)).*$'
 AnalyzeTemporaryDtors: false
 WarningsAsErrors: '*'
-CheckOptions:
```

Comment:
CheckOptions: should still be here, no? Actually we should check what lintrunner is doing with `...`

Reply:
OK